frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckUserExistsHandler.java

```diff
@@ -138,7 +138,8 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                         emailAddress,
                         IpAddressHelper.extractIpAddress(input),
                         AuditService.UNKNOWN,
-                        persistentSessionId);
+                        persistentSessionId,
+                        pair("number_of_attempts_user_allowed_to_login", incorrectPasswordCount));
```

Comment:
This looks like the wrong thing to me - if we're happy that this should reflect the total number of attempts a user had (as opposed to the number of attempts they've made), then this should be maxPasswordRetries or whatever the thing we check against here is